WRW/settings.py

```diff
@@ -0,0 +1,7 @@
+CHOICES = ('Warrior', 'Robber', 'Wizard')
+ENEEMY_LIST = ('Robber', 'Wizard', 'Warrior')
```

Comment:
Все ще не розумію, навіщо тут два однакових кортежа.

```suggestion
ENEMY_LIST = 'Robber', 'Wizard', 'Warrior'
```